training/model_management/src/azureml/model/mgmt/processors/transformers/whisper/conda.yaml

```diff
@@ -3,17 +3,32 @@ channels:
 - pytorch
 - conda-forge
 dependencies:
-- pip<=21.2.4
-- python=3.8.13
+- python=3.8.16
+- pip<=23.0.1
 - ffmpeg=4.2.2
 - pip:
-  - mlflow<3,>=2.2
+  - mlflow==2.3.1
```

Comment:
2.3.2?

Reply:
Model emitted right now contains 2.3.1 which has successful validations. Below is for t5:

![image](https://github.com/Azure/azureml-assets/assets/61145377/b65d9b39-46b5-4dbe-a574-b17b5ead3717)